### DISCUSSION TITLE
Some questions about deep potential

### DISCUSSION THREAD

**Discussion by WeJear** [deepmodeling/deepmd-kit#3734] [Q&A]
Dear developer，

I recently used dp potential to calculate some material properties, such as thermal conductivity, and I used lammps to calculate these properties. First, the data I trained came from the AIMD of vasp. I used the trained dp potential to calculate the thermal conductivity (using both the NEMD and EMD methods) and found that there was a difference with the actual calculation. Please tell me how I can solve this problem and whether I need to change a model for training  or which model is suitable for calculating the properties of materials? Hope you can give some suggestions.

Thanks。

Below is my json file and in.lammps file,
[RbReSI.json](https://github.com/deepmodeling/deepmd-kit/files/15173788/RbReSI.json)
[RbReSI-heatflux.txt](https://github.com/deepmodeling/deepmd-kit/files/15173791/RbReSI-heatflux.txt)

**Comment by wanghan-iapcm** [comment]
I suspect the training data generated by AIMD does not cover the NEMD and EMD calculations. please find https://docs.deepmodeling.com/projects/deepmd/en/r2/troubleshooting/precision.html#enough-data

**Reply by WeJear** [reply]
OK, thanks for your suggestion.

**Comment by WeJear** [comment]
I have checked the introduction of each descriptor, but I did not find any impact of these descriptors on the effect of the deep potential. I want to know will different descriptors affect the accuracy of the model and What are the differences between these descriptors?

**Reply by wanghan-iapcm** [reply]
se_e2, se_e3, dpa-1 https://arxiv.org/pdf/2304.09409
dpa-2 https://arxiv.org/abs/2312.15492

**Reply by WeJear** [reply]
OK，Thanks
